CrossMod/ItemCombining.cs

```diff
@@ -15,7 +16,7 @@ public abstract class ItemCombining : ModType {
 
 		public abstract bool CanCombine(Item item1, Item item2);
 
-		public static bool CanCombineItems(Item item1, Item item2, bool checkPrefix = true) {
+		public static bool CanCombineItems(Item item1, Item item2, bool checkPrefix = true, ConditionalWeakTable<Item, string> savedItemTagIO = null) {
```

Comment:
This new parameter doesn't seem to be used

Reply:
I also suggest re-adding the old definition and just have it call the new one in order to not break the API:
```cs
public static bool CanCombineItems(Item item1, Item item2, bool checkPrefix = true) => CanCombineItems(item1, item2, checkPrefix, null);
```

Reply:
in order to do that, I'd need to change the order so that null goes before checkPrefix, but I can do that.

Reply:
actually, I can make both args required in that case, nevermind